web/docs/migrate-from-0-11-to-0-12.md

```diff
@@ -418,14 +418,14 @@ You can follow these steps to migrate to the new auth system (assuming you alrea
     // highlight-next-line
     export const fields = defineUserSignupFields({
       address: async (data) => {
-        const address = data.address
+        const address = data.address;
```

Comment:
I'd maybe not include the semicolons if we didn't include it in other examples on this page.